backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/Setup.kt

```diff
@@ -2,10 +2,9 @@ package app.ehrenamtskarte.backend.verification.webservice
 
 import io.javalin.Javalin
 
-const val BASE_PATH = "/verification"
+const val BASE_PATH = "/verification/"
 
 fun setupApp(app: Javalin) {
-    val cardHandler = CardHandler()
-    app.post("$BASE_PATH/add-card", cardHandler::addCard)
-    app.post("$BASE_PATH/verify", cardHandler::verifyCard)
+    val graphQLHandler = VerificationGraphQLHandler()
+    app.post(BASE_PATH) { ctx -> graphQLHandler.handle(ctx) }
```

Comment:
In my opinion ctx should no be given to graphQLHandler, because it handles business logic and should not deal with api stuff. Mas consider to lift up getPayload out of GraphQLHandler and then pass the Map directly into GraphQLHandler.

At least this better testable because the Handler component is decoupled from the API.

Reply:
Hm, I just copied that from the already existing GraphQLHandler, but I will think about it 👍🏽

Reply:
The GraphQLHandler handles a http request to the graphql api. It's the interface between the inner GraphQL engine and the webservice, so imo this is just fine